docs/resources/user.md

```diff
@@ -0,0 +1,30 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mke_user Resource - terraform-provider-mke"
+subcategory: ""
+description: |-
+  User resource
+---
+
+# mke_user (Resource)
+
+User resource
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the user
+- `password` (String, Sensitive) The password of the user
+
+### Optional
+
+- `full_name` (String) The full name of the user
+- `is_admin` (Boolean) Is the user an admin
+
+### Read-Only
+
+- `id` (String) Identifier
```

Comment:
This was missing from the previous commit. `go generate` had not been run locally, and I guess that the GH action doesn't discover missing files.